android/app/src/main/java/org/uwaterloo/subletr/pages/home/list/components/DateFilter.kt

```diff
@@ -0,0 +1,185 @@
+package org.uwaterloo.subletr.pages.home.list.components
+
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.DatePickerDefaults
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.rememberDateRangePickerState
+import androidx.compose.material3.rememberModalBottomSheetState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.unit.dp
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+import org.uwaterloo.subletr.R
+import org.uwaterloo.subletr.pages.createlisting.DateInputButton
+import org.uwaterloo.subletr.pages.createlisting.DatePickerBottomSheet
```

Comment:
These sorts of components that you're using from other screens can probably be moved into the `components` directory.

Reply:
Yeah, I was thinking about that. Will do.